Pipfile

```diff
@@ -22,6 +22,7 @@ retrying = "*"
 maya = "*"
 PyJWT = "*"
 requestsdefaulter = "*"
+numpy = "*"
```

Comment:
If we dont have numpy issues anymore, can we remove this?

Reply:
Unfortunately not yet: python-slim 3.6 and 3.8 support Debian Buster, but don't yet support Debian Bullseye, the one with a NumPy > 1.1.16.